ws-nextjs-app/pages/[service]/live/[id]/README.md

```diff
@@ -0,0 +1,52 @@
+# Live Page
+
+## Description
+
+This is the first page to be created through our new NextJS app. It uses several NextJS features, including server side rendering and NextJS' pages router. 
+
+## How routes are handled
+
+Our Live Page is served through the following route: 
+`/pages/[service]/live/[id]/[[variant]]`, where the double brackets in `[[variant]]` means that the variant is optional.
+
+This route matches the file structure of this repo: 
+  📦ws-nextjs-app
+   ┗ 📂pages
+   ┃ ┣ 📂[service]
+   ┃ ┃ ┣ 📂live
+   ┃ ┃ ┃ ┗ 📂[id]
+   ┃ ┃ ┃ ┃ ┗ 📜[[...variant]].page.tsx
+   ┃ ┃ ┗ 📜[[...]].page.tsx
+   ┃ ┣ 📜_app.page.tsx
+   ┃ ┗ 📜_document.page.tsx
```

Comment:
Just checking the layout of this in the Readme. It appears on 1 line for me when its rendered, rather than in a pseudo folder-tree layout like in the actual Markdown.